eitprocessing/data_collection/__init__.py

```diff
@@ -1,19 +1,25 @@
 from __future__ import annotations
 
+import warnings
 from typing import TYPE_CHECKING, Generic, TypeVar
 
+from eitprocessing.continuous_data import ContinuousData
+from eitprocessing.eit_data import EITData
 from eitprocessing.mixins.equality import Equivalence
+from eitprocessing.sparse_data import SparseData
```

Comment:
I think a87c671ac6c81624967355ff545e69984012334b fixes it.

Reply:
I still get circular imports (which i don't if I revert to not having any constraints on `V`). The commit you refer to touches another module than this (`eit_data/__init__.py` rather than `data_collection/__init__.py`)

Reply:
I missed that one. I think it is now solved in b993ef2774100f330c663360e984fc9c12344af9. It's not pretty, but this is the result of interdependence between classes.